### PR TITLE
Fix #20: Document regeneration of TLS certs

### LINKED ISSUE #20: --force should force redownloading the TLS certificates
_From @bexelbie on November 23, 2015 8:12_

_Copied from original issue: projectatomic/vagrant-adbinfo#35_

### PR DESCRIPTION
**Pull request by brgnepal**
Fix #20 

Updated doc as per [this comment](https://github.com/projectatomic/vagrant-service-manager/issues/20#issuecomment-221831195).

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -118,7 +118,8 @@ Displays connection information for all active services in the box in a
 manner that can be evaluated in a shell. +
 If a `service` is specified, only the information for that service is displayed.
 The supported services are: Docker, OpenShift. +
-When `--script-readable` is specified the output is in `key=value` format.
+When `--script-readable` is specified the output is in `key=value` format. +
+The TLS certificates will be regenerated if it doesn't exists or found invalid.
```
Comment (hferentschik) [inline README.adoc]: "if they don't exist or found invalid"

**Comment by brgnepal** [comment]
@hferentschik Fixed

**Comment by hferentschik** [comment]
@budhrg have a look at pull #301. I tried to be a bit more verbose. WDYT?

**Comment by brgnepal** [comment]
Closing this PR as #301 looks good.

**Pull request closed without merging**